docker-compose/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ version: "2"
 
 services:
     alfresco:
-        image: alfresco/alfresco-content-repository:6.1.0
+        image: alfresco/alfresco-content-repository:latest
```

Comment:
I think we need to discuss the possibility of doing EA or RC release with new "transformer" code in.

Reply:
Yes, that would be great - maybe 6.2.0-EA1 to get the ball rolling ?